dapr-spring/dapr-spring-cloudconfig/src/test/java/io/dapr/spring/boot/cloudconfig/CloudConfigTestApplication.java

```diff
@@ -0,0 +1,14 @@
+package io.dapr.spring.boot.cloudconfig;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+
+@SpringBootApplication
+@ComponentScan(basePackages = "io.dapr.spring.boot.cloudconfig.config")
```

Comment:
@lony2003 here Spring usually creates an annotation to enable a feature.. is there no Spring Cloud annotation to enable access to all config features?

Reply:
Yeah, actually it doesn't depends on any Spring Cloud features, it can be a dependent library cause it's just a core of SpringBoot feature. Named Cloud Config just because this feature is usually bandled with other Spring Cloud components and requires an rpc connection